tests/integration_tests/performance/test_snapshot_ab.py

```diff
@@ -108,6 +108,7 @@ def sample_latency(
             values.append(value)
             fcmetrics.stop()
             microvm.kill()
```

Comment:
Should we also remove the chroot in addition or does this cover the majority of the disk space?

Reply:
I went back in our commit history, and apparently we used to delete the chroot, but it was causing problems where other parts of the framework were still expecting the chroot itself to exist. So I went ahead and only deleted the snapshot for this, which is indeed the largest contributor (everything else is at most a handful of MB)